animepahe-dl.sh

```diff
@@ -62,6 +67,13 @@ set_args() {
             r)
                 _ANIME_RESOLUTION="$OPTARG"
                 ;;
+            t)
+                _PARALLEL_RUN="true"
+                _PARALLEL_JOBS="$OPTARG"
+                [[ ${_PARALLEL_JOBS} -gt 0 ]] || {
```

Comment:
Please use `-le` and `&&` as the condition to print error message

Reply:
Using `-lt` (less than) than `-le` (less than or equal to)

Resolved snippet
```
                _PARALLEL_JOBS="$OPTARG"
                [[ ${_PARALLEL_JOBS} -lt 0 ]] && {
                    print_error "-t switch only takes a positive integer as argument."
                }
```